content/en/docs/reference/node/kernel-version-requirements.md

```diff
@@ -0,0 +1,59 @@
+---
+content_type: "reference"
+title: OS Kernel Version Requirements
+weight: 10
+---
+
+Many features rely on specific kernel functionalities and have minimum kernel version requirements.
+
+## Pod [sysctls](/docs/tasks/administer-cluster/sysctl-cluster/#safe-and-unsafe-sysctls)
+
+`sysctl` is a Linux-specific command-line tool used to configure various kernel parameters
+and it is not available on non-Linux operating systems.
+
+The following sysctls are supported in the safe set, which has a minimal kernel version requirement:
+
+- `net.ipv4.ip_local_reserved_ports` (since Kubernetes 1.27, needs kernel 3.16+);
+- `net.ipv4.tcp_keepalive_time` (since Kubernetes 1.29, needs kernel 4.5+);
+- `net.ipv4.tcp_fin_timeout` (since Kubernetes 1.29, needs kernel 4.6+);
+- `net.ipv4.tcp_keepalive_intvl` (since Kubernetes 1.29, needs kernel 4.5+);
+- `net.ipv4.tcp_keepalive_probes` (since Kubernetes 1.29, needs kernel 4.5+);
+- `net.ipv4.tcp_syncookies` (namespaced since kernel 4.6+).
+- `net.ipv4.vs.conn_reuse_mode` (used in `ipvs` proxy mode, needs kernel 4.1+);
+
+### kube proxy `nftables` proxy mode
+
+[`nftables` proxy mode](/docs/reference/networking/virtual-ips/#proxy-mode-nftables) is introduced in v1.29.
+The (alpha) nftables mode of kube-proxy now requires version 1.0.1 or later
+of the nft command-line, and kernel 5.13 or later. (For testing/development
+purposes, you can use older kernels, as far back as 5.4, if you set the
+`nftables.skipKernelVersionCheck` option in the kube-proxy config, but this is not
+recommended in production since it may cause problems with other nftables
+users on the system.
```

Comment:
This will become stale with the v1.31 release. Can we write it to be more neutral as to whether nftables support is alpha / beta / stable?

Reply:
Removed the alpha description here and only mentioned that the kernel version requiement.